Assignment6/index.html

```diff
@@ -0,0 +1,158 @@
+                    </div>
+                    <div class = "modal-body">
+                        <p>Would you like to play again?</p>
+                    </div>
+                    <div class = "modal-footer bg-dark">
+                        <button type = "button" class = "btn btn-primary" data-dismiss = "modal" id = "yes">Yes</button>
+                        <button type = "button" class = "btn btn-secondary" data-dismiss = "modal" id = "no">No</button>
+                    </div>
+                </div>
+            </div>
+        </div> 
+        
+        <script>
+        /* global $ */
+        let no;
+        // put your JavaScript code here...
+        $(document).ready(function()
+        {
+           let size = 20;
+           let value = 1;
+           //add style for rows, cols, card, and card body
+            for(let i = 0; i < Math.floor(size / 2); i++){
+                $('#main').append('<div class = "row no-gutters" style = "height:200px"><div class = "col-sm no-gutters"><div class = "card " style = "height:200px"><div class = "card-body bg-dark" value = ' + value + '></div></div></div><div class = "col-sm"><div class = "card" style = "height:200px"><div class = "card-body bg-dark" value = ' + parseInt(value + 1, 10) + '></div></div></div></div>');
+                value += 2;
+            }
+            if(size % 2 == 1){
+                $('#main').append('<div class = "row" style = "height:200px"><div class = "col-sm"><div class = "card"style = "height:200px"><div class = "card-body bg-dark"value = ' + value + '></div></div></div>');
+            }
+            let txt = "Guesses: " + 0;              
+                    $("#response").text(txt);
+            // model...
+            let model = {};
+
+            function initializeModel()
+            {
+                model.error = "";
+                model.gameStatus = "";
+                model.guesses = 0;
+                model.card;
+                model.ready;
+            }
+            
+            // view...
+            function updateView()
+            {
+               if (model.error != "")
+               {
+                    $("#response").text(model.error);
+                }
+               else
+               {
+                   if(model.gameStatus != ''){
+                    txt = "Guesses: " + model.guesses;              
+                    $("#response").text(txt);
+                    model.card.removeClass('bg-dark');
+                    model.card.addClass('bg-success');
+                    $('#my-Modal').modal('show');
+                   }else{
+                    model.card.removeClass('bg-dark');
+                    model.card.addClass('bg-danger');
+                    txt = "Guesses: " + model.guesses;              
+                    $("#response").text(txt);
+                   }
+               }
+                
+            }
+
+            //win screen...
+            $('#yes').click(function()

```

Comment:
F-1: If I win, then click outside of the modal (instead of clicking Yes or No), I continue the game, but can only guess incorrectly.